packages/client/src/components/Location/LocationSummary.tsx

```diff
@@ -18,9 +18,8 @@ import EditLocationModal from "./EditLocationModal";
 type LocationSummaryProps = {
   addressFromLocation?: any;
   isBelowInputFields?: boolean;
-  matomoTrackEvent?: Function;
-  resetChecker?: Function;
-  setActiveState?: Function;
+  resetChecker?: () => void;
+  setActiveState?: () => void;
```

Comment:
It seemed weird to me a `setSomethingFunction` doesn't accept any arguments. So I checked and setActiveState is actually not used :-/
Which brings me to another question. How to handle unused props? We can, at least, place the props directly above the component.
What do you think?
And can you double check any non-used fields in this type?

Reply:
I don't really comprehend this question. Let's discuss off-topic